### PR TITLE
add more SI units

### PR DESCRIPTION
**Pull request by NeuralSpaz**
add Power as Watt
add Electrical Capacitance as farad
add Luminous Flux as lumen
add Luminous Intensity as candela
add Energy as joule
add easy to verify tests for new units and there prefixes(eg 1GJ == GigaJoule etc...)
add picoAsString to support ElectricalCapacitance
add picoAsString test

### CODE REVIEW

File: conn/physic/units.go [hunk 1]
```diff
@@ -352,6 +352,114 @@ const (
...
+	NanoFarad  ElectricalCapacitance = 1000 * PicoFarad
+	MicroFarad ElectricalCapacitance = 1000 * NanoFarad
+	MilliFarad ElectricalCapacitance = 1000 * MicroFarad
+	Farad      ElectricalCapacitance = 1000 * MilliFarad
+	KiloFarad  ElectricalCapacitance = 1000 * Farad
+	MegaFarad  ElectricalCapacitance = 1000 * KiloFarad
+)
+
+// LuminousIntensity is a measurement of wavelength power stored as a
+// nano candela.
```
Comment (maruel) [inline conn/physic/units.go]: The comment should have a note that this is only about human visible wavelengths via a standardized luminosity function.
https://en.m.wikipedia.org/wiki/Luminosity_function
https://en.m.wikipedia.org/wiki/Photopic_vision

Something like ".. a measurement of luminous power per solid angle, on wavelengths based on 1931 CIE photopic luminosity function model of the human eye, stored as .."

File: conn/physic/units.go [hunk 2]
```diff
@@ -352,6 +352,114 @@ const (
...
+	NanoCandela  LuminousIntensity = 1
+	MicroCandela LuminousIntensity = 1000 * NanoCandela
+	MilliCandela LuminousIntensity = 1000 * MicroCandela
+	Candela      LuminousIntensity = 1000 * MilliCandela
+	KiloCandela  LuminousIntensity = 1000 * Candela
+	MegaCandela  LuminousIntensity = 1000 * KiloCandela
+	GigaCandela  LuminousIntensity = 1000 * MegaCandela
+)
+
+// LuminousFlux is a measurement of luminous flux stored as a nano lumens.
```
Comment (maruel) [inline conn/physic/units.go]: ".. a measurement of the total quantity of visible light emitted by a source, stored as ..."

**Comment by maruel** [comment]
You may want to rebase on top of origin/master to help with the PR.

File: conn/physic/units.go (hunk 1 above)
Comment (NeuralSpaz) [reply conn/physic/units.go]: I'm not sure the clarity is there in my wording.

**Comment by maruel** [comment]
gohci

File: conn/physic/units.go (hunk 1 above)
Comment (maruel) [reply conn/physic/units.go]: It's a bit verbose but it will do, I'm fine with editing later.

**Pull request merged**